frontend/src/utils/getRandomNumString.ts

```diff
@@ -0,0 +1,6 @@
+export function getRandomNumString(length = 5) {
```

Comment:
function that I've used to create simple random strings for creating ids, I could have used crypto.randomUUID but I guess that's not needed for very simple cases, and I'd imagine it being more expensive?

I don't mind just using that if people prefer it